ai_agents/devable_research_agent/.env.example

```
@@ -0,0 +1,43 @@
+# Copy to .env or ~/.config/morning-ai/.env and fill in values.
+
+MESSAGE_ENABLED=true
+ARXIV_LOOKBACK_DAYS=1
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The ARXIV_LOOKBACK_DAYS key should go before the MESSAGE_ENABLED key

(UnorderedKey)